content/articles/how-to-build-a-quiz-app-with-vanilla-javascript-and-tailwind-css/index.md

```diff
@@ -0,0 +1,399 @@
+                <button class="btn">Answer 1</button>
+                <button class="btn">Answer 2</button>
+                <button class="btn">Answer 3</button>
+                <button class="btn">Answer 4</button>
+            </div>
+        </div>
+        <div class="flex justify-center gap-4">
+            <button id="start-btn" class="bg-pink-700 px-9 py-3 text-white text-2xl rounded-lg hover:bg-pink-400">Start</button>
+            <button id="next-btn" class="bg-pink-700 px-9 py-3 text-white text-2xl rounded-lg hover:bg-pink-400">Next</button>
+        </div>
+
+        </div>
+
+    </div>
+    <script defer src="../src/script.js"></script>
+</body>
+</html>
+```
+
+In the code above, we imported several utilities from the [Tailwind CSS official website](https://tailwindcss.com/docs)  that we used in styling several components of our app. Also, the question and answer containers contain dummy code that we will later update with JavaScript.
+
+Copy and paste this CSS this code in your `style.css` file.
+
+```css
+*, *::before, *::after {
+    padding: 0;
+    margin: 0;
+    box-sizing: border-box;
+    font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
+  }
+
+  :root {
+    --hue-neutral: 200;
+    --hue-wrong: 0;
+    --hue-correct: 145;
+  }
+
+
+  .container {
+    width: 800px;
+    max-width: 80%;
+    background-color: white;
+    border-radius: 5px;
+    padding: 10px;
+    box-shadow: 0 0 10px 2px;
+  }
+
+
+  .btn {
+    --hue: var(--hue-neutral);
+    border: 1px solid hsl(var(--hue), 100%, 30%);
+    background-color: hsl(var(--hue), 100%, 50%);
+    border-radius: 5px;
+    padding: 5px 10px;
+    color: white;
+    outline: none;
+  }
+
+  .btn:hover {
+    border-color: black;
+  }
+
+  .btn.correct {
+    --hue: var(--hue-correct);
+    color: black;
+  }
+
+  .btn.wrong {
+    --hue: var(--hue-wrong);
+  }
+
+  .hide {
+    display: none;
+  }
+```
+
+The code above represents the styling for the answer buttons and their responsiveness to clicking when the user chooses an answer.
+
+Note that we also set a `hide` class to `display: none`, which we will use in JavaScript.
+
+This is how the root page of the app looks like at this point:
+
+![quiz-app]/engineering-education/how-to-build-a-quiz-app-with-vanilla-javascript-and-tailwind-css/quiz-app.jpg)
+
+### Now the JavaScript
+
+First, create a `script.js` file in your app directory. In there, you have to call the HTML elements you will work on into your JavaScript with the code below:
+```js
+const startButton = document.querySelector('#start-btn')
+const nextButton = document.querySelector('#next-btn')
+const questionContainerElement = document.querySelector('#question-container')
+const questionElement = document.querySelector('#question')
+const answerButtonsElement = document.querySelector('#answer-buttons')
+```
+Then, you have to create an array that will contain your quiz questions and their corresponding answer options.
```

Comment:
You have used `you have to` many times at this point. Try to use another phrase?